client/styled-components/Reservation.jsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { FindButton, DayButton } from './Buttons';
+import Title from './Title';
+import { DropDown, DropDownEntry } from './DropDowns';
+import CalendarInput from './CalendarInput';
+import {
+  ComponentContainer, CalendarContainer, Days, Month, GridContainer,
+} from './Divs';
+
+const cd = require('../CalendarData/CalendarData').default;
+
+class Reservation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      calendarHidden: true,
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    const { calendarHidden } = this.state;
+    this.setState({
+      calendarHidden: !calendarHidden,
+    });
+  }
+
+  render() {
+    const { calendarHidden } = this.state;
+
+    return (
+      <ComponentContainer>
+        <Title>Make a Reservation</Title>
+        <CalendarInput
+          type="text"
+          onClick={this.handleClick}
+          value={cd.currentDay}
+          readOnly
+        />
+        {calendarHidden ? <br />
+          : (
+            <CalendarContainer>
+              <Month>
+                <time dateTime={cd.OctoberMonthValue}>October 2019</time>
+              </Month>
+              <GridContainer>
+                {cd.dayAbbreviations.map((day) => <Days key={day}>{day}</Days>)}
+              </GridContainer>
+              <GridContainer>
+                {cd.October.map((day) => (day.firstDay === '01' ? <DayButton firstChild key={day.dateTimeDay}><time dateTime={day.day}>{day.day}</time></DayButton> : <DayButton key={day.dateTimeDay}><time dateTime={day.dateTimeDay}>{day.day}</time></DayButton>))}
```

Comment:
This line is a little cumbersome. Can it be refactored to multiple lines, or did the style-guide not allow it?
day.firstDay === '01' **?**
<DayButton firstChild etc..... >
  <time></time>
</DayButton> **:**
<DayButton firstChild etc..... >
  <time></time>
</DayButton

Reply:
Yeah , I think this could be refactored. Going to try to do that later on. Thinking of moving the calendar component and the logic to a separate file, then just importing the component itself to put here. Thanks for pointing it out though.